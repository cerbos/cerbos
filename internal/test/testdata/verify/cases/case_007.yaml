description: Invalid test suite with bad value for option now
