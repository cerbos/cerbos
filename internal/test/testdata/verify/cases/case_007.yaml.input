-- suite_test.yaml --
---
name: TestSuite
description: Tests for verifying something
options:
  now: "blah"
tests:
  - name: John and his leave request
    input:
      principals:
        - john
      resources:
        - john_leave_request
      actions:
        - view:public
        - approve
    expected:
      - principal: john
        resource: john_leave_request
        actions:
          view:public: EFFECT_ALLOW
          approve: EFFECT_DENY
