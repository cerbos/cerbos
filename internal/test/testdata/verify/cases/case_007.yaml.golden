{
  "suites":  [
    {
      "file":  "suite_test.yaml",
      "name":  "Unknown",
      "summary":  {
        "overallResult":  "RESULT_ERRORED"
      },
      "error":  "failed to load test suite: failed to unmarshal JSON: proto:Â (line 1:84): invalid google.protobuf.Timestamp value \"blah\""
    }
  ],
  "summary":  {
    "overallResult":  "RESULT_ERRORED"
  }
}